modules/persistent-storage-csi-smb-cifs-provision-static.adoc

```diff
@@ -0,0 +1,216 @@
+[id="persistent-storage-csi-smb-cifs-provision-static_{context}"]
+= Static provisioning
+
+You can use static provisioning to create a persistent volume (PV) and persistent volume claim (PVC) to consume existing Server Message Block protocol (SMB) shares:
+
+.Prerequisites
+* Access to the {product-title} web console.
+* {FeatureName} CSI Driver Operator and driver installed.
+* You have installed the SMB server and know the following information about the server:
+** Hostname
+** Share name
+** Username and password
+
+.Procedure
+
+To set up static provisioning:
+
+. Create a Secret for access to the Samba server using the following command with the following example YAML file:
++
+[source,terminal]
+--
+$ oc create -f <file_name>.yaml
+--
++
+[source,yaml]
+.Secret example YAML file
+--
+apiVersion: v1
+kind: Secret
+metadata:
+  name: smbcreds <1>
+  namespace: samba-server <2>
+stringData:
+  username: <username> <3>
+  password: <password> <4>
+--
+<1> Name of the Secret for the Samba server.
+<2> Namespace for the Secret for the Samba server.
+<3> Username for the Secret for the Samba server.
+<4> Password for the Secret for the Samba server.
+
+. Create a PV by running the following command with the following example YAML file:
++
+[source,terminal]
+----
+$ oc create -f <pv_file_name>.yaml <1>
+----
+<1> The name of the PV YAML file.
++
+.Example PV YAML file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  annotations:
+    pv.kubernetes.io/provisioned-by: smb.csi.k8s.io
+  name: <pv_name> <1>
+spec:
+  capacity:
+    storage: 100Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: ""
+  mountOptions:
+    - dir_mode=0777
+    - file_mode=0777
+  csi:
+    driver: smb.csi.k8s.io
+    volumeHandle: smb-server.default.svc.cluster.local/share/ <2>
+    volumeAttributes:
+      source: //<hostname>/<shares> <3>
+    nodeStageSecretRef:
+      name: <secret_name_shares> <4>
+      namespace: <namespace> <5>
+----
+<1> The name of the PV.
+<2> `volumeHandle` format: {smb-server-address}.{sub-dir-name}.{share-name}. Ensure that this value is unique for every share in the cluster.
+<3> The Samba server must be installed somewhere and reachable from the cluster with <hostname> being the hostname for the Samba server and <shares> the path the server is configured to have among the exported shares.
+<4> The name of the Secret for the shares.
+<5> The applicable namespace.
+
+. Create a PVC:
+
+.. Create a PVC by running the following command with the following example YAML file:
++
+[source, terminal]
+----
+$ oc create -f <pv_file_name>.yaml <1>
+----
+<1> The name of the PVC YAML file.
++
+.Example PVC YAML file
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: <pvc_name> <1>
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: <storage_amount> <2>
+  volumeName: <pv_name> <3>
```

Comment:
```suggestion
  storageClassName: ""
  volumeName: <pv_name> <3>
```
We also need add the storageClassName at here.

Reply:
Done.